Memerator/dist/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+   "author": {
+      "name": "Chew",
+      "id": "476488167042580481"
+   },
+   "url": "memerator.me",
+   "description": {
+      "Memerator": "Memerator is a up and coming meme website where you can post and rate other's memes."
+   },
+   "service": "Memerator",
+   "version": "1.0.0",
+   "logo": "https://cdn.memerator.me/memerator.png",
+   "thumbnail": "https://memerator.me/scott_stretch.png",
```

Comment:
Upload these images to another website like imgur, so they cant be changed.

Reply:
Imgur doesn't allow hotlinking images on sites other than forums or their own, see https://twitter.com/imgur/status/879408405403664384. Should I upload it to github and use that link?

Reply:
You just upload to imgur and then copy image url